pkg/kubecost/asset.go

```diff
@@ -2534,6 +2534,7 @@ type AssetSet struct {
 	sync.RWMutex
 	aggregateBy []string
 	assets      map[string]Asset
+	FromSource  string
```

Comment:
It doesn't look like this made it into the codec, is that of concern?

Reply:
This has been addressed. Thanks!